keyboards/binepad/bnr1/readme.md

```diff
@@ -0,0 +1,15 @@
+# BINEPAD BNR1
+
+![BINEPAD BNR1](https://imgur.com/gjLADOK.jpg)
+
+A 3% macropad.
+
+* Keyboard Maintainer: [BINEPAD]
+* Hardware Supported: BNR1 PCB (ATmega32U4)
+* Hardware Availability: [Interest Check](https://www.binepad.com/bnR1)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```